src/js/views/Dashboard/ActivityFeed/OpsLogEntry.jsx

```diff
@@ -0,0 +1,95 @@
+import { DateTime } from 'luxon'
+import Gravatar from 'react-gravatar'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+
+function OpsLogEntry({ entry }) {
+  const { t, i18n } = useTranslation()
+
+  const recordedAt = DateTime.fromISO(entry.recorded_at)
+  const changeType = entry.change_type.toLowerCase()
+  const environment = entry.environment
+  const project = entry.project_name
+  const description = entry.description
+  const version = entry.version
+  const displayName = entry.display_name
+
+  const fields = Object.entries(entry)
+    .filter(
+      ([key, value]) =>
+        value && ['project_name', 'description', 'version'].includes(key)
+    )
+    .map(([key, value]) => key)
+
+  let i18nKey
+  if (fields.includes('project_name')) {
+    if (fields.includes('version')) {
+      i18nKey = 'dashboard.activityFeed.opsLogProjectVersion'
+    } else {
+      i18nKey = 'dashboard.activityFeed.opsLogProject'
+    }
+  } else if (fields.includes('description')) {
+    if (fields.includes('version')) {
+      i18nKey = 'dashboard.activityFeed.opsLogDescriptionVersion'
+    } else {
+      i18nKey = 'dashboard.activityFeed.opsLogDescription'
+    }
+  } else if (fields.includes('version')) {
+    i18nKey = 'dashboard.activityFeed.opsLogVersion'
+  } else {
+    i18nKey = 'dashboard.activityFeed.opsLog'
+  }
+
+  return (
+    <li>
+      <Link
+        to={`/ui/operations-log?v=${entry.id}`}
+        className="flex p-2 space-x-3">
+        <Gravatar
+          className="h-8 w-8 rounded-full"
+          default="mp"
+          email={entry.email_address}
+          size={22}
+        />
+        <div className="min-w-0 flex-1 text-sm text-gray-700">
+          <Trans i18nKey={i18nKey} i18n={i18n} t={t}>
+            <span className="font-medium text-gray-700">{{ displayName }}</span>
+            <span>{{ description }}</span>
+            <span>{{ changeType }}</span>
+            <Link
+              to={`/ui/projects/${entry.project_id}`}
+              className="font-medium text-blue-700 hover:text-blue-800">
+              {{ project }}
+            </Link>
```

Comment:
This generates malformed HTML since the link on line 47 cannot contain another link.  I've done this in the past by registering a click handler on the outer link and then disabling event bubbling IIRC.... check out how Edit and Delete work in https://github.com/AWeber-Imbi/imbi-ui/blob/main/src/js/components/Table/Row.jsx